skglm/tests/test_lbfgs_solver.py

```diff
@@ -36,5 +38,46 @@ def test_lbfgs_L2_logreg():
     )
 
 
+@pytest.mark.parametrize("use_efron", [True, False])
+def test_L2_Cox(use_efron):
+    try:
+        from lifelines import CoxPHFitter
+    except ModuleNotFoundError:
+        pytest.xfail(
+            "Testing L2 Cox Estimator requires `lifelines` packages\n"
+            "Run `pip install lifelines`"
+        )
+
+    alpha = 10.
+    n_samples, n_features = 50, 10
+    random_state = 0
+
+    tm, s, X = make_dummy_survival_data(
+        n_samples, n_features, normalize=True, with_ties=use_efron,
+        random_state=random_state)
+
+    datafit = compiled_clone(Cox(use_efron))
+    penalty = compiled_clone(L2(alpha))
+
+    datafit.initialize(X, (tm, s))
+    w, *_ = LBFGS().solve(X, (tm, s), datafit, penalty)
+
+    # fit lifeline estimator
+    stacked_tm_s_X = np.hstack((tm[:, None], s[:, None], X))
+    df = pd.DataFrame(stacked_tm_s_X)
+
+    estimator = CoxPHFitter(penalizer=alpha, l1_ratio=0.)
+    estimator.fit(
+        df, duration_col=0, event_col=1,
+    )
+    w_ll = estimator.params_.values
+
+    p_obj_skglm = datafit.value((tm, s), w, X @ w) + penalty.value(w)
+    p_obj_ll = datafit.value((tm, s), w_ll, X @ w_ll) + penalty.value(w_ll)
+
+    np.testing.assert_allclose(w, w_ll, atol=1e-3)
```

Comment:
use rtol here if possible, it's more explicit. atol=1e-5 could be very large or very small

Reply:
with ``rtol``, I can't get below ``1e-1``because of https://github.com/CamDavidsonPilon/lifelines/pull/1534 and other still-to-investigate factors.